src/web.rs

```diff
@@ -327,3 +327,17 @@ pub fn switch_host(host: String) -> Promise {
         Ok(JsValue::UNDEFINED)
     })
 }
+
+#[wasm_bindgen]
+pub fn ln_create_wallet() -> Promise {
+    set_panic_hook();
+
+    future_to_promise(async move {
+        match crate::create_wallet().await {
```

Comment:
let's do `crate::lightning::create_wallet`, and then export only `lightning` and not `lightning::create_wallet`. Then we can namespace the other methods later into their own modules.

Reply:
I have it like that before this PR, I don't know why I changed 🥲